d2g.go

```diff
@@ -113,7 +113,7 @@ func podmanCopyArtifacts(containerName string, payload *dockerworker.DockerWorke
 }
 
 func env(env map[string]string) map[string]string {
-	return env
+	return map[string]string{}
```

Comment:
Do you envision we'll ever do anything in this transform helper? If not, shall we just remove it and set generic worker env to be `map[string]string{}`?

Reply:
That's a very good question. There is a reasonable chance we'll need it when we implement missing features (interactive shell / or video/audio loopback stuff) although I'm not sure. Perhaps let's leave it in for now, as it is at least a placeholder and reminds us that no env vars are being set at all.